x-pack/plugins/secops/public/components/page/hosts/hosts_details_panel/index.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiDescriptionList,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiPanel,
+  EuiTitle,
+} from '@elastic/eui';
+import React from 'react';
+import { pure } from 'recompose';
+import { HostsEdges } from 'x-pack/plugins/secops/server/graphql/types';
+import { getOrEmpty } from '../../../empty_value';
+import * as i18n from './translations';
+
+interface OwnProps {
+  data: HostsEdges[];
+  loading: boolean;
+}
+
+type HostDetailsPanelProps = OwnProps;
+
+export const HostDetailsPanel = pure<HostDetailsPanelProps>(({ data, loading }) => (
+  <EuiFlexItem style={{ maxWidth: 600 }}>
+    <EuiPanel>
+      <EuiTitle size="s">
+        <h3>{i18n.HOST}</h3>
+      </EuiTitle>
+
+      <EuiHorizontalRule margin="xs" />
+      <EuiDescriptionList listItems={getHostDetailsItems(data[0])} type="column" compressed />
```

Comment:
Have you considered making the content of this widget draggable to the timeline?

Reply:
I had not! Happy to make the content draggable to the timeline. Some fields have multiple values (like IP w/ IPv4 & IPV6) -- is it fine to make each individual item draggable or should they be grouped? I don't see anywhere we're doing grouping, and that seems less granular in terms of letting the user choose whether they want to AND / OR them as well.

Reply:
Summary values are now draggable! If a summary value happens to be an array, a draggable is created for each item. This will still need a slight cleanup before submitting in light of https://github.com/elastic/ingest-dev/issues/223